packages/mui-material/src/utils/getChildRef.js

```diff
@@ -0,0 +1,10 @@
+import * as React from 'react';
+
+export default function getChildRef(children) {
```

Comment:
In practice, it only works with a child?

```suggestion
export default function getChildRef(child) {
```

Reply:
renamed -> https://github.com/mui/material-ui/pull/42818/commits/5bc21fab32b7ab14c9abab8dcf802a68b7c2cf00